### PR TITLE
Improve tab icons in left panel

### LINKED ISSUE #5269: UX: Sidebar icons improvements
The change to using icons for labeling the sidebar in #5117 makes sense, but the icons can be a bit confusing, especially for those that do not have English as a native language:
- The [palette icon](https://github.com/ellisonbg/jupyterlab/blob/f7e0d69363b288bea6c97e8884897ad5892b93fa/packages/theme-dark-extension/style/icons/md/ic_palette_24px.svg) strikes me as more related to themeing. Without knowing what a palette is, it can be very confusing. A replacement icon would preferably be more tied up to the *command* concept.
- The ["running" icon](https://github.com/ellisonbg/jupyterlab/blob/35df45add8437e888003de1bad2cf3fdb518844e/packages/theme-light-extension/style/icons/md/ic_directions_run_24px.svg) is based on a homonym (moving fast, and an process under execution). This concept does not necessarily translate to other languages. A possible suggestion is multiple cogwheels (multiple, since a single cogwheel indicates settings, by convention).

cc @ellisonbg

### PR DESCRIPTION
**Pull request by gnestor**
Fixes #5269

To do:

* [ ] Agree on better icons for left panel tabs
* [x] Improve icon handling in lab so that icon color doesn't need to be hard-coded in SVGs (I'm going to create a new issue for this)

Here are some ideas:

* Commands
  * A [menu icon](https://material.io/tools/icons/?search=menu&icon=menu&style=baseline)
  * A [command line prompt icon](https://material.io/tools/icons/?search=arrow&icon=arrow_forward_ios&style=baseline)
* Running: 
  * A [play icon](https://material.io/tools/icons/?search=play&icon=play_arrow&style=baseline) icon or [play circle icon](https://material.io/tools/icons/?search=play&icon=play_circle_filled&style=baseline)
  * A [stop icon](https://material.io/tools/icons/?search=stop&icon=stop&style=baseline)
  * A [fast-forward icon](https://material.io/tools/icons/?search=forward&icon=fast_forward&style=baseline)
  * A [power (on/off icon)](https://material.io/tools/icons/?search=on&icon=power_settings_new&style=baseline)
  * A [playlist icon](https://material.io/tools/icons/?icon=playlist_play&style=baseline)
  * A [servers icon](https://material.io/tools/icons/?icon=dns&style=baseline)
  * An [eject icon](https://material.io/tools/icons/?icon=eject&style=baseline)
* Cell inspector: 
  * A [dials icon](https://material.io/tools/icons/?icon=tune&style=baseline) 
  * A [settings icon](https://material.io/tools/icons/?search=setting&icon=settings&style=baseline)

One variation:

image

Another variation: 

image

### CODE REVIEW

**Comment by ian-r-rose** [comment]
My two cents:

* I like the play icon for the "Running" tab. Since the panel also serves to shut down sessions, perhaps a play/pause icon?
* The cog and slider images for the cell tools are a bit confusing, they seem to me like they might open the settings editor. I thought the wrench was okay.
* The command icons are tough. The '>' icon looks a lot like the play icon, which is confusing. The `=` icon looks kind of like the icon that is used for `@jupyterlab/toc`, and doesn't really connote commands to me. I don't know that I have a good suggestion here, though.

**Comment by gnestor** [comment]
> I like the play icon for the "Running" tab. Since the panel also serves to shut down sessions, perhaps a play/pause icon?

Play/pause is a good idea 👍

> The cog and slider images for the cell tools are a bit confusing, they seem to me like they might open the settings editor. I thought the wrench was okay.

Looks like there is consensus here...

> The command icons are tough. The '>' icon looks a lot like the play icon, which is confusing. The = icon looks kind of like the icon that is used for @jupyterlab/toc, and doesn't really connote commands to me. I don't know that I have a good suggestion here, though.

This is tough...

**Comment by t-makaro** [comment]
I like the idea of play/pause much better than just a play button, but since one can only see and shutdown stuff from the running panel, and not pause/resume it doesn't make total sense. I still like the idea of a power icon, but what about a [stop](https://material.io/tools/icons/?search=stop&icon=stop&style=baseline) icon?

**Comment by gnestor** [comment]
I like stop but the material icon for stop is just a square: https://material.io/tools/icons/?search=stop&icon=stop&style=baseline

**Comment by mangecoeur** [comment]
Nice! Some suggestions:

To avoid confusion with the play/pause button and the console why not use one in a circle:

iconfinder_ic_play_circle_outline_48px_352074

And/or use a double-chevron for the command palette

64x64

**Comment by jasongrout** [comment]
A double Chevron also reminds me of a command prompt, which is close to commands in my mind.

**Comment by t-makaro** [comment]
For the running (shutdown/stop menu), we could use a stop icon in a circle: 
image [from here](https://fontawesome.com/icons/stop-circle?style=regular)

Just spit balling:

For the command palette:

how about this [superpowers](https://fontawesome.com/icons/superpowers?style=brands) icon? The command palette is a place of super powers...
image

On that same theme: a [wand](https://fontawesome.com/icons/magic?style=solid)
image

I was also think about a "remote" icon, but I can't find a nice one that doesn't look like it is sending a signal.

Maybe the [adjust icon](https://fontawesome.com/icons/adjust?style=solid)
image

**Comment by gnestor** [comment]
@t-makaro We're currently using Material Icons so let's look for icon candidates in that set vs. Font Awesome 👍 

I like the [remote control icon](https://material.io/tools/icons/?search=remote&icon=settings_remote&style=baseline) for command palette but Material Icons' version is not the best...

I updated the description with suggetions from here and https://github.com/jupyterlab/jupyterlab/issues/5269.

**Comment by ellisonbg** [comment]
I will have a look at this and give some background on how the existing icons were chosen.

**Comment by ellisonbg** [comment]
Catching up...

* Last summer, we really struggled with the command palette icon. We probably spent 20-30 hours on it. The challenge is that our notion of a "command" is very abstract, and all our attempts to find an icon to represent a command, were too concrete and misleading (too focused on a particular type of action, like code). The idea of the palette was that "here is a set of things you can use in your work." But I agree the palette icon doesn't have a connotation of being somehow related to visual effect or color. I will look through the MD icons again, but this one is really tough.
* For the running panel. I like the ideas being explored here, with some combination of a play or stop button, but a circle around it. The idea I probably like the best is a stop button with a circle around it - as it best matches the action you can take in the panel. I didn't see it in the MD icon set, but we can create it manually with little effort in Figma is there is support for the idea.
* I think we should leave the others as is (cell tools, etc.)

**Comment by gnestor** [comment]
I've updated this branch so that it only changes the running icon. We can iterate on other sidebar icons. 

We have 2 options, the current branch is using "stop circle" icon.

Stop circle icon:
image

Remove circle icon:
image

**Comment by jasongrout** [comment]
Is there still more to do on this PR? I notice the WIP in the title was removed.

**Comment by gnestor** [comment]
This current branch replaces the running icon with the stop circle icon (which seems like an improvement to me). It's ready to merge 👍

File: packages/application/style/icons.css [hunk 1]
```diff
@@ -213,6 +213,10 @@
   background-image: var(--jp-icon-run);
 }
 
+.jp-RunningIcon {
+  background-image: var(--jp-icon-remove-circle);
```
Comment (jasongrout) [inline packages/application/style/icons.css]: remove-circle?

File: packages/theme-dark-extension/style/urls.css [hunk 2]
```diff
@@ -87,6 +87,7 @@
   --jp-icon-r-selected: url('icons/jupyter/r_selected.svg');
   --jp-icon-r: url('icons/jupyter/r.svg');
   --jp-icon-refresh: url('icons/md/ic_refresh_18px.svg');
+  --jp-icon-remove-circle: url('icons/md/stop-circle-solid.svg');
```
Comment (jasongrout) [inline packages/theme-dark-extension/style/urls.css]: it seems that remove-circle here is confused with stop-circle

**Comment by jasongrout** [comment]
Thanks. It seems that there is confusion in the final changes between remove-circle and stop-circle. Can we take out the remove-circle icon if we aren't using it, and change all the wording to stop-circle?

**Comment by ian-r-rose** [comment]
Can we also remove the CSS classes and SVGs for the "running person" icon?

**Comment by ian-r-rose** [comment]
Otherwise, I think we should move forwards with this.

**Comment by gnestor** [comment]
Done!

**Comment by jasongrout** [comment]
Thanks!

**Pull request merged**